### PR TITLE
Implement city branding

### PR DESCRIPTION
**Pull request by paylakatel**
This PR:
- updates the app UI to have the city branding
- as a result it:
     - removes some components - Modal.js, FeatureCounts.js
     - adds a "About" page
     - moves the navigation and footer to the Layout component

### CODE REVIEW

File: components/Filters.js [hunk 1]
```diff
@@ -49,25 +49,30 @@ export default function Filters(props) {
...
+            className="w-25 font-weight-bold"
           >
             All
           </Button>
         </ButtonGroup>
-        <Form className="m-2">
+        <Form className="m-1">
           <FormGroup>
-            <Label for="from">From Date:</Label>
+            <Label for="from" className="font-weight-bold text-uppercase">
```
Comment (fionawhim) [inline components/Filters.js]: Should be `htmlFor` instead of `for`, here and below.

File: components/Layout.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 export default class Layout extends React.Component {
   render() {
     return (
-      // create div for page content with height of the entire veiw port
       <div>
         <Head>
           <title> {this.props.title} </title>
+          <link
+            rel="stylesheet"
+            href="https://www.boston.gov/crispus/css/public.css"
```
Comment (fionawhim) [inline components/Layout.js]: Should pull from `patterns.boston.gov`: `<link rel="stylesheet" type="text/css" href="https://patterns.boston.gov/css/public.css" />`

File: components/Layout.js [hunk 3]
```diff
@@ -26,20 +30,182 @@ export default class Layout extends React.Component {
...
-
         {/* Make sure the heigh of the body element is the entire view port */}
         <style global jsx>{`
-          @import url('https://fonts.googleapis.com/css?family=Roboto:400,600');
-          body {
-            font-family: 'Roboto';
-            min-height: 100vh;
-            position: relative;
-            margin: 0;
+          @import url('https://fonts.googleapis.com/css?family=Lora:400,400i,700|Montserrat:400,700');
```
Comment (fionawhim) [inline components/Layout.js]: Can remove this once you switch the stylesheet URL.

File: components/Layout.js [hunk 4]
```diff
@@ -26,20 +30,182 @@ export default class Layout extends React.Component {
...
+          }
+          .form-control {
+            border-radius: 0;
+            border-color: #d2d2d2;
+          }
+        `}</style>
+        {/* set container div with room for navbar  */}
+        <div style={{ minHeight: 'calc(100vh - 125px)' }}>
+          <Navbar>
+            <NavbarBrand href="http://www.visionzeroboston.org/" className="">
```
Comment (fionawhim) [inline components/Layout.js]: Can you remove `className`?

File: pages/about.jsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+              This map and associated dataset, provided as part of the Vision
+              Zero Boston program, contains records of the date, time, location,
+              and type of crash for incidents requiring public safety response
+              which may involve injuries or fatalities. All records are compiled
+              by the Department of Innovation and Technology from the City’s
+              Computer-Aided Dispatch (911) system and verified as having
+              required a response from a public safety agency. To protect the
+              privacy of individuals involved in these incidents, we do not
+              indicate the severity of specific crashes or whether medical care
+              was provided in any specific case.{' '}
```
Comment (fionawhim) [inline pages/about.jsx]: Can remove this `{' '}` and I think a few others.

**Review by fionawhim — CHANGES_REQUESTED** [review]

File: components/Filters.js (hunk 1 above)
Comment (paylakatel) [reply components/Filters.js]: Updated - what's the difference between the two?

File: components/Layout.js (hunk 2 above)
Comment (paylakatel) [reply components/Layout.js]: 👍

File: components/Layout.js (hunk 3 above)
Comment (paylakatel) [reply components/Layout.js]: 👍

File: components/Layout.js (hunk 4 above)
Comment (paylakatel) [reply components/Layout.js]: 👍

File: pages/about.jsx (hunk 5 above)
Comment (paylakatel) [reply pages/about.jsx]: 👍

File: components/Filters.js (hunk 1 above)
Comment (fionawhim) [reply components/Filters.js]: React uses `htmlFor` because `for` is a reserved word in JavaScript. Likewise with `className` vs. `class`.

**Review by fionawhim — APPROVED** [review]

File: components/Filters.js (hunk 1 above)
Comment (paylakatel) [reply components/Filters.js]: ohhh nice

**Pull request merged**